### PR TITLE
[CodeDeploy] Added `getDeployment` method

### PR DESCRIPTION
**Pull request by martinssipenko**
Added `getDeployment()`method to `CodeDeployClient`.

### CODE REVIEW

**Comment by martinssipenko** [comment]
@jderusse `Generated output not changed` fails because the newly added `getDeployment` method reuses the same classes as `createDeployment` method, but the generated output for each method is different, how should I proceed with this?

Also I'm not sure what to do with the issues reported by Psalm.

**Comment by jderusse** [comment]
> @jderusse `Generated output not changed` fails because the newly added `getDeployment` method reuses the same classes as `createDeployment` method, but the generated output for each method is different, how should I proceed with this?
> 
> Also I'm not sure what to do with the issues reported by Psalm.

run `./bin/generate --all` to let the script pick the right version for you

**Comment by martinssipenko** [comment]
@jderusse now only Psalm issue remains, should I add it to baseline?

**Comment by jderusse** [comment]
> @jderusse now only Psalm issue remains, should I add it to baseline?

Yes, it's a false positive.

**Comment by martinssipenko** [comment]
Alright, CI is green now.

File: src/Service/CodeDeploy/tests/Integration/CodeDeployClientTest.php [hunk 1]
```diff
@@ -80,6 +81,20 @@ public function testCreateDeployment(): void
...
+        $client = $this->getClient();
+
+        $input = new GetDeploymentInput([
+            'deploymentId' => 'change me',
+        ]);
+        $result = $client->getDeployment($input);
+
+        $result->resolve();
+
+        // self::assertTODO(expected, $result->getdeploymentInfo());
```
Comment (jderusse) [inline src/Service/CodeDeploy/tests/Integration/CodeDeployClientTest.php:95]: could you add an assertion to this test?

**Review by jderusse — APPROVED** [review]
Thank you for this PR. 
A last small comment, and I'll merge the PR

**Comment by martinssipenko** [comment]
@jderusse can we expect a release of this any time soon?

**Comment by jderusse** [comment]
> @jderusse can we expect a release of this any time soon?

I'm currently on it see https://github.com/async-aws/aws/pull/1140

**Pull request merged**